frontend/src/Editor/Inspector/Components/Table/ColumnManager/ColumnPopover.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import Popover from 'react-bootstrap/Popover';
+import { StylesTabElements } from './StylesTabElements';
+import { PropertiesTabElements } from './PropertiesTabElements';
+
+export const ColumnPopoverContent = ({
+  column,
+  index,
+  darkMode,
+  currentState,
+  onColumnItemChange,
+  getPopoverFieldSource,
+  setColumnPopoverRootCloseBlocker,
+  component,
+  props,
+  columnEventChanged,
+  handleEventManagerPopoverCallback,
+}) => {
+  const [activeTab, setActiveTab] = useState('propertiesTab');
+
+  const timeZoneOptions = [
```

Comment:
Move these to a comon contsant file. Also seems to be used in DatePickerProperties.js